### PR TITLE
*: Support connection_id()

### PR DESCRIPTION
**Pull request by shenli**
Fix: https://github.com/pingcap/tidb/issues/715

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -591,6 +592,9 @@ func CreateSession(store kv.Storage) (Session, error) {
 	variable.BindSessionVars(s)
 	variable.GetSessionVars(s).SetStatusFlag(mysql.ServerStatusAutocommit, true)
 
+	// set connection id
+	s.SetValue(builtin.ConnectionIDKey, s.sid)
```
Comment (siddontang) [inline session.go]: connection id may be in tidb server conn's ID?

Comment (shenli) [inline session.go]: Returns the connection ID (thread ID) for the connection. Every connection has an ID that is unique among the set of currently connected clients.
SessionID is unique among the clients.

**Comment by siddontang** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**